client/layout/masterbar/item.jsx

```diff
@@ -49,6 +52,7 @@ class MasterbarItem extends Component {
 				onTouchStart={ this.preload }
 				onMouseEnter={ this.preload }
 			>
+				{ this.props.hasUnseen && <span className="masterbar__item-bubble" /> }
```

Comment:
It would be worth including an `aria-label` (either on this element itself or the wrapping element) when we're rendering these so the unread status is available to screen-reader users.  Github's notifications indicator is a good example.

Reply:
Updated, thanks for taking the time